sources/packages/backend/libs/test-utils/src/factories/application.ts

```diff
@@ -80,7 +84,10 @@ export async function saveFakeApplicationDisbursements(
   },
   options?: {
     applicationStatus?: ApplicationStatus;
+    offeringIntensity?: OfferingIntensity;
     createSecondDisbursement?: boolean;
+    disbursementScheduleStatusA?: DisbursementScheduleStatus;
```

Comment:
I would recommend `firstDisbursementInitialValues:Partial<DisbursementScheduleStatus>, secondDisbursementInitialValues:Partial<DisbursementScheduleStatus>` to allow more initial values to passed if required.